docs/getting-started/index.md

```diff
@@ -3,52 +3,55 @@ title: Getting started
 section: getting-started
 ---
 
-Developers should have some prior knowledge of the [Shopify app ecosystem](https://shopify.dev/concepts/apps). Currently, Shopify App CLI creates apps using either [Node.js](https://nodejs.org/) or [Ruby on Rails](https://rubyonrails.org/).
+Developers should have some prior knowledge of the [Shopify app ecosystem](https://shopify.dev/concepts/apps). Shopify App CLI creates apps using either [Node.js](https://nodejs.org/) or [Ruby on Rails](https://rubyonrails.org/).
 
-## Install
+## Requirements
 
-Shopify App CLI installs using a shell script. Download and run it in your terminal with one command:
+- [Ruby](https://www.ruby-lang.org) 2.5.1+ 
+- [Node.js](https://nodejs.org) 10.0.0+
+- [curl](https://curl.haxx.se) 7.0.0+
+- A [Shopify partner account](https://partners.shopify.com/signup)
+- A [Shopify development store](https://help.shopify.com/en/partners/dashboard/development-stores#create-a-development-store) to install and test apps
 
-### Mac OS and Ubuntu
```

Comment:
Is "MacOS and Ubuntu" not important? Is the tool compatible with other flavours of Linux?

Reply:
Good question! I think this is kind of a matter of what we “officially support” versus “it might work but you're on your own”. It seems pretty likely to me that the CLI works on other Linux flavors but we just lack the bandwidth to test or support them.

I just didn't like that the Mac/Ubuntu and Windows install directions were identical, except with additional system requirements for windows. So I wanted to unify the install step and get more explicit with the system requirements.

Perhaps this means we need to add a section on officially supported operating systems or something similar.

Reply:
@tanema Can you verify what OS's we support?

Reply:
macOS, Debian distributions and windows through the WLS